components/feed-card/index.tsx

```diff
@@ -33,6 +37,18 @@ const LQDFeedCard = ({ feed, isDetailPage, handleCommentPress }: FeedCard) => {
 
   const handleLikePress = () => {
     console.log('Like');
+    const previousHasLiked = hasLikedFeed;
+    const previousLikeCount = likeCount;
+
+    setHasLikedFeed((prev) => !prev);
+    setLikeCount((prev) => (hasLikedFeed ? prev - 1 : prev + 1));
+
+    likeMutation.mutate(id, {
+      onError: () => {
+        setHasLikedFeed(previousHasLiked);
+        setLikeCount(previousLikeCount);
+      },
+    });
```

Comment:
Do we really need to do the .mutate functionality here? Also, the idea of tanstack is that we don't need a local state for isLiked and likeCount 

When you target and update this strategy in the store, it'll automatically update on the screen